keda/Chart.yaml

```diff
@@ -8,11 +8,11 @@ kubeVersion: ">=v1.23.0-0"
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 2.12.0
+version: 2.12.1
```

Comment:
Can you please revert these changes and remove the new version?

Reply:
Could you explain why? That's what is explained in Contributing.md

Reply:
That is just overall docs, not every PR is supposed to do that. I'll update the docs next week